components/TrackList.vue

```diff
@@ -109,14 +109,15 @@ export default Vue.extend({
     this.tracks = tracks;
   },
   methods: {
+    /**
+     * @param {number} ticks The number of ticks to convert to track length
+     * @returns {string} Returns the length of the track in the format XX:XX
+     */
     getRuntime(ticks: number) {
-      function padLeft(string: string, pad: string, length: number): string {
```

Comment:
This was meant to pad cases where we have a single-digit number of seconds. Afaik, without this, we'll get durations like `2:5`. Was this checked?

Reply:
Ahh I couldn't work out what it did :p. That makes sense. Reverted 👍